src/patternfly/components/Check/examples/Check.md

```diff
@@ -8,97 +8,62 @@ cssPrefix: pf-v5-c-check
 ## Examples
 ### Basic
 ```hbs
-{{#> check}}
-  {{#> check-input check-input--attribute='id="check-basic" name="check-basic"'}}{{/check-input}}
-  {{#> check-label check-label--attribute='for="check-basic"'}}Check{{/check-label}}
-{{/check}}
+{{> check check-input--id="basic-example" check-label--text="Check"}}
 ```
 
 ### Required
 ```hbs
-{{#> check check--IsRequired="true"}}
-  {{#> check-input check-input--attribute='id="check-required" name="check-required"'}}{{/check-input}}
-  {{#> check-label check-label--attribute='for="check-required"'}}Check{{/check-label}}
-{{/check}}
+{{> check check-input--id="check-required" check-label--text="Check" check--IsRequired=true}}
 ```
 
 ### Checked
 ```hbs
-{{#> check}}
-  {{#> check-input check-input--attribute='id="check-checked" name="check-checked" checked'}}{{/check-input}}
-  {{#> check-label check-label--attribute='for="check-checked"'}}Check checked{{/check-label}}
-{{/check}}
+{{> check check-input--id="check-checked" check-label--text="Check checked"}}
 ```
 
 ### Label wrapping input
 ```hbs
-{{#> check check--type="label" check--attribute='for="check-label-wrapping-input"'}}
-  {{#> check-input check-input--attribute='id="check-label-wrapping-input" name="check-label-wrapping-input"'}}{{/check-input}}
-  {{#> check-label check-label--type="span"}}Check label wraps input{{/check-label}}
-{{/check}}
+{{> check check-input--id="check-label-wrapping-input" check-label--text="Check label wraps input" check--IsLabelWrapped=true}}
```

Comment:
This example is generating `id="-input"` on the parent check container since that just looks for `check--id` but you're setting `check-input--id` on all of the examples https://github.com/patternfly/patternfly/blob/99476bba3b3433847d6c68f83dffb9a943b29803/src/patternfly/components/Check/check.hbs#L8-L10

Looks like the radio `isLabelWrapped` example works because you're setting `radio--id` on that specific example instead of `radio-input--id`

I'm not sure I understand the id-helper hbs entirely, but at a glance I'd probably expect setting the check--id on the outer container and being able to generate all of the inner id's from check--id? If I get the idea of the helper, I wonder if this would work:

```
{{> check check--id="some-id"}}

// check.hbs
{{#> check--context check-input--id=(ternary check-input--id (concat check-input--id "-input") (concat check--id "-input)}}
  // all the existing stuff in check.hbs
{{/check-context}}
```

Reply:
@mcoker Since this is a workspace issue, why don't we create a followup?